src/sentry/integrations/slack/message_builder/issues.py

```diff
@@ -58,6 +58,18 @@
 from sentry.utils import json
 
 STATUSES = {"resolved": "resolved", "ignored": "ignored", "unresolved": "re-opened"}
+PROVIDER_TO_COMMIT_LINK_URL_FORMAT = {
+    "github": "{base_url}/commit/{commit_id}",
+    "integrations:github": "{base_url}/commit/{commit_id}",
+    "integrations:github_enterprise": "{base_url}/commit/{commit_id}",
+    "visualstudio": "{base_url}/commit/{commit_id}",  # TODO: buggy
+    "integrations:vsts": "{base_url}/commit/{commit_id}",
+    "gitlab": "{base_url}/commit/{commit_id}",  # TODO: buggy
+    "integrations:gitlab": "{base_url}/commit/{commit_id}",  # TODO: buggy
```

Comment:
these three provider ids i get serialization errors if i use them in tests, are these outdated? this list of providers is based off that of `commitLink.tsx`